config/v1.2/aws-k8s-cni.yaml

```diff
@@ -0,0 +1,130 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+# kubernetes versions before 1.8.0 should use rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: aws-node
+rules:
+- apiGroups:
+  - crd.k8s.amazonaws.com
+  resources:
+  - "*"
+  - namespaecs
```

Comment:
@liwenwu-amazon I realize this is already merged, but it looks like theres a typo in this ClusterRole rule: `namespaecs` -> `namespaces`, though with the wildcard above it this line may be redundant.